packages/muelu/test/toggletransfer/parameters_semi_tent_kokkos.xml

```diff
@@ -0,0 +1,96 @@
+<ParameterList name="MueLu">
+
+  <!-- Configuration of the Xpetra operator (fine level) -->
+  <ParameterList name="Matrix">
+    <Parameter name="number of equations"                   type="int" value="1"/> <!-- Number of PDE equations at each grid node.-->
+  </ParameterList>
+
+  <!-- Factory collection -->
+  <ParameterList name="Factories">
+
+    <!-- Note that ParameterLists must be defined prior to being used -->
+
+    <!-- =======================  PART I  ======================= -->
+
+    <ParameterList name="myTentativePFact1">
+      <Parameter name="factory"                             type="string" value="SemiCoarsenPFactory_kokkos"/>
+      <Parameter name="semicoarsen: coarsen rate" type="int" value="3"/>
```

Comment:
This parameter seems a bit small compared to values used in actual runs, should it be pushed up to test this more realistically?

Reply:
I think this test (the same as parameters_semi_tent.xml) is set to a small parameter to test the functionality of multiple semi coarsening levels. If I increase the coarsen rate, it might get reduced to a single level. We do end up using one level most of the time but I think multiple levels is more general for testing. I could add another test with a higher coarsen rate though.